src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.MonthCalendarAccessibleObject.cs

```diff
@@ -569,20 +568,37 @@ internal CalendarTodayLinkAccessibleObject TodayLinkAccessibleObject
         {
             get
             {
+                MonthCalendar? owningMonthCalendar;
                 SelectionRange? range;
 
                 switch (CalendarView)
                 {
                     case MONTH_CALDENDAR_MESSAGES_VIEW.MCMV_MONTH:
-                        range = _owningMonthCalendar.SelectionRange;
+                        range = this.TryGetOwnerAs(out owningMonthCalendar) ? owningMonthCalendar.SelectionRange : new SelectionRange();
 
                         return DateTime.Equals(range.Start.Date, range.End.Date)
                             ? $"{range.Start:D}"
                             : $"{range.Start:D} - {range.End:D}";
                     case MONTH_CALDENDAR_MESSAGES_VIEW.MCMV_YEAR:
-                        return $"{_owningMonthCalendar.SelectionStart:y}";
+                        if(this.TryGetOwnerAs(out owningMonthCalendar))
+                        {
+                            return $"{owningMonthCalendar.SelectionStart:y}";
+                        }
+                        else
+                        {
+                            return string.Empty;
+                        }
+
```

Comment:
Nit: `else` is not needed here and in similar places below.
```suggestion
                        if (this.TryGetOwnerAs(out owningMonthCalendar))
                        {
                            return $"{owningMonthCalendar.SelectionStart:y}";
                        }
                        
                        return string.Empty;
```